atomicapp/nulecule/container.py

```diff
@@ -0,0 +1,54 @@
+import os
+import shutil
+import subprocess
+import uuid
+import logging
+
+from atomicapp.constants import APP_ENT_PATH
+
+logger = logging.getLogger(__name__)
+
+
+class DockerHandler(object):
+    """Interface to interact with Docker."""
+
+    def __init__(self, docker_cli='/usr/bin/docker'):
+        self.docker_cli = docker_cli
+
+    def pull(self, image, update=False):
+        if not self.is_image_present(image) or update:
+            logger.debug('Pulling Docker image: %s' % image)
+            pull_cmd = [self.docker_cli, 'pull', image]
```

Comment:
At some point we need to pull the `docker://` part off of the image names that are in the Nulecule like is done currently in https://github.com/projectatomic/atomicapp/blob/master/atomicapp/utils.py#L138 

I don't know if I like having  `docker://` prepended. I suspect we will need to change that at some point in the future.. Should we put a XXX comment in there to revisit this?